go.mod

```diff
@@ -66,7 +66,7 @@ require (
 	github.com/onsi/gomega v1.7.1 // indirect
 	github.com/opentracing/basictracer-go v1.0.0
 	github.com/opentracing/opentracing-go v1.1.0
-	github.com/pkg/errors v0.8.1
+	github.com/pkg/errors v0.9.1
```

Comment:
Why we upgraded this module on the way? It looks like irrelevant change.

Reply:
Don't know when this was changed. I thought it was some automatic dependency resolution.

Reply:
Let's remove then (:

Reply:
Removed

Reply:
This is actually needed (: